src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx

```diff
@@ -0,0 +1,759 @@
+      } else {
+        addSubscription(
+          queryArchivedRecordings(connectUrl)
+            .pipe(
+              first(),
+              map((v) => v.data.archivedRecordings.data as ArchivedRecording[])
+            )
+            .subscribe({
+              next: (recordings) => {
+                if (recordings.length > 0) {
+                  handleArchivedRecordings(recordings);
+                } else {
+                  handleStateErrors(NO_RECORDINGS_MESSAGE);
+                }
+              },
+              error: (err) => {
+                handleStateErrors(err.message);
+              },
+            })
+        );
+      }
+    },
+    [
+      addSubscription,
+      context.reports,
+      categorizeEvaluation,
+      queryArchivedRecordings,
+      handleArchivedRecordings,
+      handleStateErrors,
+      setUsingCachedReport,
+      setReportTime,
+      setReport,
+      setIsLoading,
+    ]
+  );
+
+  const generateReport = React.useCallback(() => {
+    addSubscription(
+      context.target
+        .target()
+        .pipe(
+          filter((target) => target !== NO_TARGET),
+          first()
+        )
+        .subscribe((target) => {
+          handleLoading();
+          setTargetConnectURL(target.connectUrl);
+          dispatch(automatedAnalysisAddTargetIntent(target.connectUrl));
+          addSubscription(
+            queryActiveRecordings(target.connectUrl)
+              .pipe(
+                first(),
+                tap((resp) => {
+                  if (resp.data == undefined) {
+                    if (isGraphQLAuthError(resp)) {
+                      context.target.setAuthFailure();
+                      throw new Error(authFailMessage);
+                    } else {
+                      throw new Error(resp.errors[0].message);
+                    }
+                  }
+                }),
+                map((v) => v.data.targetNodes[0].recordings.active.data[0] as Recording),
+                tap((recording) => {
+                  if (recording === null || recording === undefined) {
+                    throw new Error(NO_RECORDINGS_MESSAGE);
+                  }
+                })
+              )
+              .subscribe({
+                next: (recording) => {
+                  context.reports
+                    .reportJson(recording, target.connectUrl)
+                    .pipe(first())
+                    .subscribe({
+                      next: (report) => {
+                        setReport(recording.name);
+                        categorizeEvaluation(report);
+                        setIsLoading(false);
+                      },
+                      error: (_) => {
+                        handleStateErrors(FAILED_REPORT_MESSAGE);
+                      },
+                    });
+                },
+                error: (err) => {
+                  if (isAuthFail(err.message)) {
+                    context.target.setAuthRetry();
```

Comment:
Not sure we need this as auth modal will call this if credentials are entered. So, this one is stale.